packages/modules/pricing/integration-tests/__tests__/services/pricing-module/price-list.spec.ts

```diff
@@ -562,7 +568,6 @@ moduleIntegrationTestRunner<IPricingModuleService>({
                   currency_code: "EUR",
                 }),
                 expect.objectContaining({
-                  rules_count: 0,
```

Comment:
It was locally initialised in MikroORM entity and we do not have local initialised values in DML